app/views/families/_family.html.erb

```diff
@@ -9,5 +9,5 @@
   <td><%= family.animals %></td>
   <td><%= family.active %></td>
   <td><%= link_to t("families.index.destroy"), family_path(family), method: :delete, class: "list-button" %></td>
-  <td><%= link_to t("families.index.edit"), family_path(family), class: "list-button" %></td>
+  <td><%= link_to t("families.index.edit"), edit_family_path(family), method: :get, class: "list-button" %></td>
```

Comment:
I believe `method: :get` is the default and does not need to be explicitly set.

Reply:
I removed it, thanks!